prow/cmd/deck/static/prow/prow.ts

```diff
@@ -651,6 +651,7 @@ function redraw(fz: FuzzySearch): void {
     }
     drawJobHistogram(totalJob, jobHistogram, now - (12 * 3600), now, max);
     if (rerunStatus != null) {
+        gtag("event", "troll_rerun_popup", {event_category: "engagement", transport_type: "beacon"});
```

Comment:
I don't think we should log this event from the `redraw` function so that redrawing the page (due to refresh or filter change) doesn't cause the event to be logged multiple times. Can we do this when the button is clicked instead like we do for the ProwJob YAML button? https://github.com/kubernetes/test-infra/blob/a1a4c9fdf83b7157f89231f4e27fa7abb028060f/prow/cmd/deck/static/prow/prow.ts#L700